projects/common/src/lib/controls/lcu-map/lcu-map.component.ts

```diff
@@ -861,6 +859,15 @@ export class LcuMapComponent implements OnInit, OnDestroy, OnChanges, AfterViewI
     this._currentMapModel.Longitude = loc.Longitude;
     // this.DisplayMarkerInfo(loc);
   }
+/**
+ * called when the user has selected a new location to add to their journey and adds activity
+ * to the last activity group
+ * @param event 
+ */
+  public addIconClicked(event) {
+    this._displayedJourney.ActivityGroups[this._displayedJourney.ActivityGroups.length - 1].Activities.push(event);
```

Comment:
I think here we should be using this.DisplayedJourney instead of this._displayedJourney in order to take advantage of the get method and keep consistent.